lib/polint.rb

```diff
@@ -129,36 +168,18 @@ def check_pair(key, val, contexts, is_fuzzy)
       @errors  += 1 if not_in_val.any? || not_in_key.any? || is_empty
       @fuzzies += 1 if is_fuzzy
 
-      if key =~ /(\d+):(.*)/
-        lineno = $1
-        file_and_line = "#{@pofile}:#{$1}:"
```

Comment:
No, not losing it, I just moved this into the `log` function (and added `error` and `warn` helpers) as I needed to call it from outside `check_pair` to do some higher level validations such as the number of plurals.

The output should be exactly the same as before.